<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AgroChemLab v.0.1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeView" name="treeView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="filterButton">
          <property name="text">
           <string>Filter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableView" name="mainTable"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="agroButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Agro &gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ecoButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Eco &gt;&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Работа</string>
    </property>
    <addaction name="openDatabase"/>
    <addaction name="newDb"/>
    <addaction name="separator"/>
    <addaction name="exitApplication"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справочники</string>
    </property>
    <addaction name="open_SMOList"/>
    <addaction name="open_OwnerList"/>
    <addaction name="open_RegionList"/>
    <addaction name="separator"/>
    <addaction name="open_TipList"/>
    <addaction name="open_KulturaList"/>
    <addaction name="separator"/>
    <addaction name="open_IndexList"/>
    <addaction name="open_GranulList"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Отчеты</string>
    </property>
    <addaction name="agroHim"/>
    <addaction name="open_Tables"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Расчеты</string>
    </property>
    <addaction name="raschUdob"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newDb"/>
   <addaction name="openDatabase"/>
   <addaction name="separator"/>
   <addaction name="agroHim"/>
   <addaction name="open_Tables"/>
   <addaction name="separator"/>
   <addaction name="raschUdob"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openDatabase">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/png/FolderOpened_Yellow.png</normaloff>:/png/FolderOpened_Yellow.png</iconset>
   </property>
   <property name="text">
    <string>Открыть базу данных</string>
   </property>
  </action>
  <action name="exitApplication">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/png/Shutdown_Box_Red.png</normaloff>:/png/Shutdown_Box_Red.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="open_SMOList">
   <property name="text">
    <string>СМО</string>
   </property>
  </action>
  <action name="open_OwnerList">
   <property name="text">
    <string>Землепользователи</string>
   </property>
  </action>
  <action name="open_IndexList">
   <property name="text">
    <string>Индекс почв</string>
   </property>
  </action>
  <action name="open_GranulList">
   <property name="text">
    <string>Гранулометрический состав</string>
   </property>
  </action>
  <action name="open_RegionList">
   <property name="text">
    <string>Районы</string>
   </property>
  </action>
  <action name="open_TipList">
   <property name="text">
    <string>Типы полей</string>
   </property>
  </action>
  <action name="open_KulturaList">
   <property name="text">
    <string>Культуры</string>
   </property>
  </action>
  <action name="open_Tables">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/png/Magnifier2.png</normaloff>:/png/Magnifier2.png</iconset>
   </property>
   <property name="text">
    <string>Таблицы</string>
   </property>
  </action>
  <action name="newDb">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/png/Plus__Orange.png</normaloff>:/png/Plus__Orange.png</iconset>
   </property>
   <property name="text">
    <string>Создать базу данных</string>
   </property>
  </action>
  <action name="agroHim">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/png/1406827132_ordering.png</normaloff>:/png/1406827132_ordering.png</iconset>
   </property>
   <property name="text">
    <string>Агрохимическая характеристика</string>
   </property>
  </action>
  <action name="raschUdob">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/png/1406827353_icontexto-green-01.png</normaloff>:/png/1406827353_icontexto-green-01.png</iconset>
   </property>
   <property name="text">
    <string>Удобрений</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
